gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Thu Oct 12 17:50:39 PDT 2017
 distributionBase=GRADLE_USER_HOME
```

Comment:
Can the **gradle/wrapper/gradle-wrapper.jar** file be downloaded as part of a build? Does it need to reside in SCM?

Reply:
Needs to be in SCM. This is how the gradle wrapper works; the ./gradlew script calls the gradle-wrapper.jar which will download gradle if not already installed. It allows for anyone to checkout the project and run ./gradlew build without having to first install gradle.

Reply:
ACK @fgreg